modules/serverless-workflow-tag-revisions.adoc

```diff
@@ -0,0 +1,46 @@
+// Module is included in the following assemblies:
+//
+// serverless/knative-client.adoc
+
+[id="tag-revisions_{context}"]
+= Tag revisions 
+
+A tag in a traffic block of service creates a custom URL, which points to a referenced revision. A user can define a unique tag for an available revision of a service which creates a custom URL by using the format `http(s)://TAG-SERVICE.DOMAIN`.
+
+A given tag must be unique to its traffic block of the service. `kn` supports assigning and unassigning custom tags for revisions of services as part of the `kn service update` command:
+
+----
+kn service update svc --tag @latest=candidate --tag svc-vwxyz=current
+----
+
+* **`--tag RevisionName=Tag`** values:
+** `--tag` flag requires two values separated by a `=`.
+** `RevisionName` string refers to name of the `Revision`.
+** `Tag` string denotes the custom tag to be given for this Revision.
+** Use the identifier `@latest` for the RevisionName to refer to the latest ready revision of the service, you can use this identifier only once with `--tag` flag.
+** If the `service update` command is updating the Configuration values for the Service (along with tag flags), `@latest` reference will be pointed to the created Revision after applying the update.
+** `--tag` flag can be specified multiple times.
+** `--tag` flag may assign different tags to the same revision
+
+
+[NOTE]
```

Comment:
If this note applies to the last bullet in the list above, then I would put a "+" on the blank line above the note to keep it indented properly with that bullet. If it's just another generic piece of information they should know, then maybe just add it as another bullet in this list?